### DISCUSSION TITLE
GitHub plans and protected branches

### DISCUSSION THREAD

**Discussion by collimarco** [community/community#72725] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am working on some projects alone. I would like to disable force push for additional security.

But the problem is with GitHub plans...

I have some projects on my personal account and other projects grouped in a Organization account (I am the only member of this organization).

If I buy a Team plan for my personal account, can I also apply the protected branches on the repositories inside that Organization? Or I would need to buy *another* Team subscription for that Organization?

**Comment by softwaredevelop** [comment]
GitHub Free for organizations - branch rules cannot be assigned to private repositories.
[GitHub Free for organizations](https://docs.github.com/en/get-started/learning-about-github/githubs-plans#github-free-for-organizations)

GitHub Team - branch rules can be assigned to private repositories.
[GitHub Team](https://docs.github.com/en/get-started/learning-about-github/githubs-plans#github-team)

**Reply by collimarco** [reply]
Ok, but when I click Upgrade on Github it asks me if I want to buy the Team plan for my personal account or for my organization.

If I buy the Team plan for my personal account, can I use the Team features *also* on my Organization?

**Reply by collimarco** [reply]
Basically I would like to use "protected branches" both on my personal repositories and on the repositories of the organization.

What should I do? Should I buy 2 plans (even if it's just me)?

**Reply by softwaredevelop** [reply]
I have a free personal account. I also created an organization from it, which is also free.

I haven't created a separate organization yet.

The GitHub documentation explains personal accounts and organizations as separate types.
https://docs.github.com/en/get-started/learning-about-github/types-of-github-accounts

I asked ChatGPT, and according to it, if you need a Team plan organization, you should request the Team upgrade for the organization. If you create an organization from a Team-upgraded personal account, it will be free.

So, I'm guessing blindly that you should upgrade the organization to a Team plan. Good luck!

**Reply by collimarco** [reply]
Ok, but if I upgrade only the Organization, then I cannot use protected branches on repos on my personal account...

**Reply by softwaredevelop** [reply]
In my opinion...

If you create a repo with a personal account, you can use branch protections according to the personal account plan.

For organizations, it seems you can set up _repository rulestets_ separately, but that requires an enterprise plan.
https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-organization-settings/managing-rulesets-for-repositories-in-your-organization
